sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc

```diff
@@ -1,18 +1,18 @@
 [id="sandboxed-containers-4-9-release-notes"]
-= {sandboxed-containers-first} {product-version} release notes
+= {sandboxed-containers-first} 1.1 release notes
 include::modules/common-attributes.adoc[]
 :context: sandboxed-containers-release-notes
 
 toc::[]
 
-[id="ocp-4-9-about-this-release"]
+[id="sandboxed-containers-1-1-about-this-release"]
 == About this release
 
-These release notes track the development of {sandboxed-containers-first} in Red Hat {product-title}.
+These release notes track the development of {sandboxed-containers-first} 1.1 alongside Red Hat {product-title} {product-version}.
 
-This product is currently in Technology Preview. {sandboxed-containers-first} is not intended for production use. For more information, see the Red Hat Customer Portal link:https://access.redhat.com/support/offerings/techpreview[support scope] for features in Technology Preview.
```

Comment:
This removed sentence shows up in a few places in the collection and seems like relevant information for the user. I'd consider keeping it.

Reply:
> This removed sentence shows up in a few places in the collection and seems like relevant information for the user. I'd consider keeping it.

The product manager requested I remove it.

Reply:
The product manager requested I remove it.